### PR TITLE
Various adjustments

### PR DESCRIPTION
**Pull request by RazvanDH**
- addressed this issue: https://github.com/RazvanDH/rs-golden-ratio-grid/issues/2
- added `$golden-grid-vendor-prefixes` flag for autoprefixer / manual prefixes
- updated breakpoints mixin to use the latest version
- removed default padding applied on fallback
- `clear: both` pseudo class should now actually work :feelsbadman:

### CODE REVIEW

File: index.scss [hunk 1]
```diff
@@ -138,38 +138,73 @@ $breakpoints-ie8: false!default;
...
+    } @else {
+      $max-breakpoint: $max-breakpoint;
+    }
+  }
+
+  @return $max-breakpoint;
+}
+
 
 @mixin breakpoint($string) {
```
Comment (igoratron) [inline index.scss:154]: why is this here and not in the other project?

File: index.scss [hunk 2]
```diff
@@ -138,38 +138,73 @@ $breakpoints-ie8: false!default;
...
 
 
 // config
 $golden-grid-max-number: 10!default;
-$golden-grid-fallback-class: '.ie-8'!default;
+$golden-grid-fallback-class: '.no-flexbox'!default;
 $golden-grid-padding: 1.5rem!default;
 $golden-grid-ratio: 1.618!default;
 $golden-grid-utility-classes: true!default;
+$golden-grid-vendor-prefixes: true!default;
```
Comment (igoratron) [inline index.scss:207]: I think these options need some docs in the readme or somewhere

File: index.scss [hunk 3]
```diff
@@ -256,24 +292,28 @@ $golden-scale: golden-scale(1);
 
   @if gr(no-flex-class) {
     #{unquote(gr(no-flex-class))} & {
-      > * {
-        @include column-padding;
-      }
-
       &:after {
-        content: '';
+        display: table;
```
Comment (igoratron) [inline index.scss]: seems like there are reasons to go with `block`, thoughts? http://cssmojo.com/the-very-latest-clearfix-reloaded/

**Comment by igoratron** [comment]
LGTM

File: index.scss (hunk 1 above)
Comment (RazvanDH) [inline index.scss:154]: this is the compiled version. will make it automated https://github.com/RazvanDH/rs-golden-ratio-grid/issues/4

File: index.scss (hunk 2 above)
Comment (RazvanDH) [inline index.scss:207]: will add a pretty documentation soon
